apps/etterlatte-samordning-vedtak/.nais/dev-api.yaml

```diff
@@ -0,0 +1,180 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: etterlatte-api
+  namespace: etterlatte
+  labels:
+    team: etterlatte
+spec:
+  image: "{{image}}"
+  port: 8080
+  ingresses:
+    - "https://etterlatte-samordning-vedtak.ekstern.dev.nav.no"  # Skru av etterhvert, skal nås via etterlatte-gw
```

Comment:
Tillatter nais at to ulike apper har samme ingress?

Reply:
Kom vel frem til at vi vil ha nye ingresser uansett https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5313/commits/a9377d6f67d0b3694ac661bcfadbee7d01167711

Reply:
Trur to appar med samme ingress er oppskrift på trøbbel på eit eller anna tidspunkt, så fint å unngå det uansett.

Reply:
det vil jo være en overgangsperiode her, men når vi kommer til tidspunktet hvor ingressen faktisk må flyttes så må man jo ha det - men det bør jo la seg gjøre å svitsje over i løpet av ikke altfor mange minutter